### PR TITLE
OCPBUGS-44010: added skip test for vsphere ccm jobs.

### PR DESCRIPTION
**Pull request by vr4manta**
[OCPBUGS-44010](https://issues.redhat.com/browse/OCPBUGS-44010)

### Changes
- Added a skip test to exclude the UDP traffic LB test

### Notes
Due to this being Platform: External, the existing vsphere skip settings is not being honored.  So for this test, we are skipping these tests manually.

### CODE REVIEW

**Comment by vr4manta** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-external-vsphere-ccm

**Comment by vr4manta** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-external-vsphere-ccm

**Comment by vr4manta** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-external-vsphere-ccm

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.18.yaml [hunk 1]
```diff
@@ -469,7 +469,9 @@ tests:
     env:
       TEST_ARGS: --disable-monitor=service-type-load-balancer-availability
       TEST_SKIPS: provisioning should mount multiple PV pointing to the same storage
```
Comment (mtulio) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.18.yaml:471]: To improve the management, I wonder if we can consolidate those envs in the workflow `upi-vsphere-platform-external-ccm` as we'll need this in all jobs using it, wdyt?

Comment (vr4manta) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.18.yaml:471]: For this removal, its only failing in 4.18 at the moment.  I can look into moving some of the other args as well if it makes sense.  Let me investigate and i'll return with my findings.  Thanks!

**Comment by vr4manta** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-external-vsphere-ccm

**Comment by vr4manta** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-external-vsphere-ccm

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.18.yaml (hunk 1 above)
Comment (vr4manta) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.18.yaml:471]: I'll do a separate PR on moving these.  It should be fine to skip the tests in older OCP version tests.

**Comment by vr4manta** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-external-vsphere-ccm

**Comment by vr4manta** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-external-vsphere-ccm

**Comment by vr4manta** [comment]
/assign @vrutkovs 
Ready for review / approval.  Thanks!

**Comment by vrutkovs** [comment]
I don't see any explanation why the test started failing

**Comment by vr4manta** [comment]
@vrutkovs , Since its platform External (not vsphere), that would be my guess.  We are running the CCM with Platform: External as a test for these. just using vSphere as the underlying platform to test the various code paths.

**Comment by vrutkovs** [comment]
oh, okay, so this test may pass on External cloud controller, but since we're using vsphere it won't work in this case, right?

**Comment by vr4manta** [comment]
Correct.  When it runs, it fails just like it does when its on normal vSphere IPI / UPI.  It should be skipped, but I'm not sure if setting skip for all External platforms makes sense.  Maybe that would be a good alternative to research in future.

**Comment by vrutkovs** [comment]
Okay, I see, thanks!

/approve

**Pull request merged**